cmd/server_test.go

```diff
@@ -762,6 +764,7 @@ write-git-creds: true
 	Equals(t, "override-my-hostname", passedConfig.TFEHostname)
 	Equals(t, "override-my-token", passedConfig.TFEToken)
 	Equals(t, false, passedConfig.WriteGitCreds)
+	Equals(t, "override-status-name", passedConfig.StatusName)
```

Comment:
You also need to add `status-name: status` to line 679 to match the rest of the test